invenio_github/api.py

```diff
@@ -347,7 +361,12 @@ def get_user_repositories(self):
         repos = deepcopy(self.user_available_repositories)
         if repos:
             # 'Enhance' our repos dict, from our database model
-            for repo in self.user_enabled_repositories:
+            db_repos = Repository.query.filter(
+                Repository.github_id.in_(
+                    [int(k) for k in self.user_available_repositories.keys()]
```

Comment:
We were filtering repos based on `self.user_enabled_repositories`, which means "repositories enabled locally by the user". That was wrong